src/search-filters/components/DomainsPopup.js

```diff
@@ -0,0 +1,153 @@
+import React, { PureComponent } from 'react'
+
+// import ReactDOM from 'react-dom'
+import PropTypes from 'prop-types'
+import { connect } from 'react-redux'
+import { bindActionCreators } from 'redux'
+import styles from './DomainsPopup.module.css'
+import { actions, selectors } from '../'
+import { selectors as sidebar } from '../../overview/sidebar-left'
+// import image from '../../assets/search-solid.svg';
+// import { FilteredRow } from './'
+
+import IndexDropdownSB from './IndexDropdownSB'
+
+class DomainsPopup extends PureComponent {
+    static propTypes = {
+        // filteredTags: PropTypes.arrayOf(PropTypes.string).isRequired,
+        // addTagFilter: PropTypes.func.isRequired,
+        // showTagFilter: PropTypes.func.isRequired,
+        // delTagFilter: PropTypes.func.isRequired,
+        // tagFilterDropdown: PropTypes.bool.isRequired,
+        isSidebarOpen: PropTypes.bool.isRequired,
+        checkCount: PropTypes.func.isRequired,
+        checkDomains: PropTypes.func.isRequired,
+        // hideTagFilter: PropTypes.func.isRequired,
+        // suggestedTags: PropTypes.arrayOf(PropTypes.string).isRequired,
+        // toggleFilterTypes: PropTypes.func.isRequired,
+        // showfilteredTypes: PropTypes.bool.isRequired,
+        filteredDomains: PropTypes.arrayOf(PropTypes.object).isRequired,
+        addDomainFilter: PropTypes.func.isRequired,
+        // showDomainFilter: PropTypes.func.isRequired,
+        domainFilterDropdown: PropTypes.bool.isRequired,
+        hideDomainFilter: PropTypes.func.isRequired,
+        delIncDomainFilter: PropTypes.func.isRequired,
+        delExcDomainFilter: PropTypes.func.isRequired,
+        // isExclusive: PropTypes.bool.isRequired,
+        // fetchSuggestedDomains: PropTypes.func.isRequired,
+        suggestedDomains: PropTypes.arrayOf(PropTypes.object).isRequired,
+    }
+
+    renderFilteredDomains = () => {
+        return !this.props.domainFilterDropdown
```

Comment:
This can also be changed to 
```suggestion
        return !this.props.domainFilterDropdown && 
```